tests/nn_test.py

```diff
@@ -314,6 +314,19 @@ def testVarianceScalingBatchAxis(self):
 
     self.assertEqual(shape, jnp.shape(val))
 
+  def testVarianceScalingError(self):
+    rng = random.PRNGKey(0)
+    shape = (5,)
+    initializer = nn.initializers.variance_scaling(
+      scale=1.0, mode='fan_avg', distribution='truncated_normal')
+
+    with self.assertRaisesRegex(
+      ValueError,
+      f"Can't compute input and output sizes of a 1"
```

Comment:
```suggestion
      "Can't compute input and output sizes of a 1"
```
to fix the flake8 error